lib/game_runner.rb

```diff
@@ -3,7 +3,8 @@
 require 'player_builder'
 require 'prompt'
 require 'game'
-require 'board'
```

Comment:
I see you still have the old `board.rb` file. Is it being used anywhere? If not, are you planning on removing it in the future? Typically you wouldn't want to leave unused code in a codebase - it can make it very challenging for people new to the project, and even for yourself a few months down the line. 

_Occasionally_ it can make sense to separate out the work to remove dead code with a separate ticket/PR. But you should typically rely on the git history if you ever need to revert something/view past code.

Also as a separate note, it's not a concern with this project as you aren't interacting with other systems, (and I don't believe you have introduced issues here). But when adding new features you want to make everything backwards compatible to limit [regressions](https://www.gratasoftware.com/what-is-regression-in-software-development/#:~:text=Regressions%20are%20a%20type%20of,regression%20is%20usually%20bug%20fixes.). What if another system relied on the file/Class being named 'Board' and you modify it to be 'Lite3Board'. That other system would need to be changed to continue to work. 

I guess what I'm getting at, is to reflect on why you introduced a new class, rather than modifying the existing board class.

Reply:
@wynspeare I resolved the board.rb file in the next PR. 

My original idea was to create two separate types board and game classes since they behave a bit differently. 

I think maybe I should create a base game, and board class similar to how I did the player class. I'll definitely follow your advice to creating more flexible classes in the future. 

I also need to get in the habit of having a working game, before I make a PR request  😄.  I'm just trying to keep them smaller.